.devcontainer/postStartCommand.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export DEBIAN_FRONTEND=noninteractive
+yes | pip install -e .
```

Comment:
It might happen when conflict appear. I will try without `yes` but keep `DEBIAN_FRONTEND=noninteractive` as it's usefull for possible update of the script (apt install for example)

Reply:
I usually do `apt install -y` for that, it's less verbose.